packages/loader/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@gql2ts/loader",
+  "private": false,
+  "version": "1.0.0",
+  "description": "Webpack loader for gql2ts",
+  "main": "dist/index.js",
+  "files": [
+    "dist"
+  ],
+  "typings": "dist/index.d.ts",
+  "scripts": {
+    "prepublish": "tsc -p tsconfig.json",
+    "build:watch": "tsc -w -p tsconfig.json"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/avantcredit/gql2ts.git"
+  },
+  "keywords": [
+    "gql2ts",
+    "graphql",
+    "typescript"
+  ],
+  "author": "Brandon Herman <brandon.herman@avant.com> (https://www.avant.com)",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/avantcredit/gql2ts/issues"
+  },
+  "homepage": "https://github.com/avantcredit/gql2ts#readme",
+  "devDependencies": {
+    "@types/loader-utils": "^1.1.3",
+    "@types/memory-fs": "^0.3.1",
+    "graphql": ">= 0.10 <1.0.0",
+    "memory-fs": "^0.4.1",
+    "webpack": "^4.16.3"
+  },
+  "peerDependencies": {
+    "graphql": ">= 0.10 <1.0.0"
+  },
+  "dependencies": {
+    "@gql2ts/from-query": "^1.8.1",
+    "@gql2ts/types": "^1.8.1",
+    "graphql-tag": "^2.9.2",
+    "loader-utils": "^1.1.0",
+    "prettier": "^1.14.0"
```

Comment:
let's change `prettier` to a peerDependency and do `>=1.4.0` (when TS support was added)